app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
`CoordinatorLayout` is quite advanced, I wouldn't recommend using it here.

Do you have a sketch of what the app will look like?